sbpy/activity/gas/productionrate.py

```diff
@@ -590,7 +595,7 @@ def from_Drahus(self, integrated_flux, mol_data, ephemobj, vgas=1 * u.km/u.s,
         ...                     ephemobj, vgas, aper, b=b) # doctest: +REMOTE_DATA
 
         >>> q  # doctest: +REMOTE_DATA +FLOAT_CMP
-        <Quantity 1.09899965e+25 1 / s>
+        <MaskedQuantity 1.09899965e+25 1 / s>
```

Comment:
Interesting! I'm trying to figure out why this change happened in astropy 5 and to make it work also for older astropy versions. I'll make it a separate PR because these changes fix the failing remote tests and should be merged.

Reply:
Sounds good!